pkg/kubectl/cmd/config/get_contexts.go

```diff
@@ -65,21 +65,11 @@ func NewCmdConfigGetContexts(out io.Writer, configAccess clientcmd.ConfigAccess)
 		Long:    getContextsLong,
 		Example: getContextsExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			validOutputTypes := sets.NewString("", "json", "yaml", "wide", "name", "custom-columns", "custom-columns-file", "go-template", "go-template-file", "jsonpath", "jsonpath-file")
-			supportedOutputTypes := sets.NewString("", "name")
-			outputFormat := cmdutil.GetFlagString(cmd, "output")
-			if !validOutputTypes.Has(outputFormat) {
-				cmdutil.CheckErr(fmt.Errorf("output must be one of '' or 'name': %v", outputFormat))
-			}
-			if !supportedOutputTypes.Has(outputFormat) {
-				fmt.Fprintf(out, "--output %v is not available in kubectl config get-contexts; resetting to default output format\n", outputFormat)
-				cmd.Flags().Set("output", "")
-			}
 			cmdutil.CheckErr(options.Complete(cmd, args, out))
 			cmdutil.CheckErr(options.RunGetContexts())
 		},
 	}
-	cmdutil.AddOutputFlags(cmd)
+	cmd.Flags().StringP("output", "o", "", "Output format. Only support 'name', use other output format will reset to default output format")
```

Comment:
This is quite confusing. Since it **only** supports `name`, how would it reset to default output format?

Reply:
if you don't use ```--output``` flag